packages/react/src/hooks/index.ts

```diff
@@ -1 +1,3 @@
-export * from './useDataRef';
+export * from './useNotifications';
+export * from './usePreferences';
+export * from './useCounts';
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** we should export here the provider as well and only `useNovu` hook (not exported rn)